ts/Extensions/ScrollablePlotArea.ts

```diff
@@ -327,7 +332,23 @@ Chart.prototype.moveFixedElements = function (): void {
 
     fixedSelectors.push(axisClass as any, axisClass + '-labels');
 
-    fixedSelectors.forEach(function (className: string): void {
+    return fixedSelectors;
+};
+
+/**
+ * These elements are moved over to the fixed renderer and stay fixed when the
+ * user scrolls the chart
+ * @private
+ */
+Chart.prototype.moveFixedElements = function (): void {
+    var container = this.container,
+        fixedRenderer = this.fixedRenderer;
+    this.fixedSelectors = this.getFixedElements();
+
+    // Add the possibility to edit the list.
+    fireEvent(this, 'afterGetFixedElements');
```

Comment:
1. `fireEvent(this, 'afterGetFixedElements');` should be called in the method, so if we ever call this method, anywhere again, event will be fired too

2. Instead of defining another object/array in Chart class, could you try this? See:
```js
fireEvent(this, 'afterGetFixedElements',  { fixedSelectors });
return fixedSelectors;
```

Then in Radial axis you could do this:
```js
addEvent(Highcharts.Chart, 'afterGetFixedElements', function (event): void { 
  if (event.fixedSelectors) { /* modify event.fixedSelectors */ } 
});
```